.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature]"
+labels: enhancement
+assignees: ''
+
+---
+
+#### Is your feature request related to a problem? Please describe.
+A clear and concise description of what the problem is.
+Ex. *I want to do X, but there is no way to do it.*
+
+#### Describe the solution you'd like
+A clear and concise description of what you want to happen.
+Ex. *I think it would be nice if you would add feature Y so it will make it easier.*
+
+#### Describe alternatives you've considered
+A clear and concise description of any alternative solutions or features you've considered.
+Ex. *I considered Z, but that didn't work because...*
+
+#### Additional context
+Add any other context or screenshots about the feature request here.
+Ex. *Here's a photo of my cat!*
```

Comment:
We are at the bottom of the template, and I disagree about "a clear and concise description" everywhere.

IMO, at least one paragraph should say:
Why you'd really want this feature? Explain fow it will change your life and your NewPipe experience!

This was just an example, but the idea is to have one paragraph where people THINK about their request. Do they really WANT it, or it was just a thought they had but they don't really care about it.

And if they give good reasons why they really WANT it, we can understand better, and prioritize (dunno if it's how we say) these requests, or on the contrary, if they didn't give good reasons AND it seems like a **niche** request, maintaners / contributors will know there is more important things to fix / features to bring

Reply:
> and I disagree about "a clear and concise description" everywhere.

You would delete this sentence everywhere?

>Why you'd really want this feature? Explain how it will change your life and your NewPipe experience!

MY DEAD MOTHER COULD COME BACK! yeah I get you good point

Reply:
> You would delete this sentence everywhere?

No, I didn't meant to remove it everywhere. My bad ih you understood that

Reply:
okay, uhm. how can I understand that part then